scripts/mujoco/evaluate_dataset.py

```diff
@@ -0,0 +1,6 @@
+import minari
+
+for dataset_id in minari.list_local_datasets().keys():
+    ds = minari.load_dataset(dataset_id)
+    episodic_return = sum([ep.rewards.sum() / len(ds) for ep in ds])
+    print(f"{dataset_id} - return: {episodic_return}")
```

Comment:
I would remove this file, I guess you use it for sanity check, but we can easily write it (and it is checking all local dataset)

Reply:
Okay i.willl remove it

Reply:
moved to https://gist.github.com/Kallinteris-Andreas/237e34d745d9d5ca146e5e81fdb3a2d7